scripts/spack/configs/toss_3_x86_64_ib/packages.yaml

```diff
@@ -26,9 +26,9 @@ packages:
     buildable: false
   mvapich2:
     externals:
-    - spec: mvapich2@2.3%clang@10.0.0 process_managers=slurm arch=linux-rhel7-ivybridge
+    - spec: mvapich2@2.3%clang@10.0.0 arch=linux-rhel7-ivybridge process_managers=slurm
```

Comment:
This ordering matters?

Reply:
I'll revert this, I had to remove and then re-add this as a spack bug was found and then fixed.